e2e/test/scenarios/dashboard-filters/reproductions/29347-remapped-field-values.cy.spec.js

```diff
@@ -0,0 +1,258 @@
+import {
```

Comment:
this was added in https://github.com/metabase/metabase/pull/29391 but didn't get backported, so it's go in with this PR